pkg/controller/authentication/pgsqlcerts.go

```diff
@@ -0,0 +1,163 @@
+//
+// Copyright 2022 IBM Corporation
```

Comment:
Should the copyright statement really be 2022 here given this is newly-added code?

Reply:
addressed